plugin/src/main/kotlin/com/agoda/testing/screenshot/build/ScreenshotsPlugin.kt

```diff
@@ -40,7 +40,7 @@ class ScreenshotsPlugin : Plugin<Project> {
 
     project.afterEvaluate {
       if (screenshotExtensions.addDeps) {
-        it.dependencies.add("androidTestImplementation", "$DEPENDENCY_GROUP:$DEPENDENCY_CORE:0.11.0")
```

Comment:
I there is more code that needs to be updated for plugins like python3 support, I think this only  going to update core classes and functions on which screenshot depends not the plugin itself